### PR TITLE
Data pipeline upgrade path: Remove CTA [MERGE AFTER FULL RELEASE]

### PR DESCRIPTION
**Pull request by Pralish**
This PR should only be merged after running the OTS and manually changing "{{cms:helper render_call_to_action :id}}" cta helper with api_form helper.

### CODE REVIEW

**Comment by Pralish** [comment]
@donrestarone , This PR should only be merged after running the OTS and manually changing "{{cms:helper render_call_to_action :id}}" cta helper with api_form helper. 

Let me know what you think

**Comment by donrestarone** [comment]
@Pralish I included the OTS as a migration here: https://github.com/restarone/violet_rails/pull/236/files#diff-22bdc0282cc9f8117de455983ef0639cfc22da575ba7a6cf5d1d077f45dcec1f

what are your thoughts?

File: app/services/analytics_report_service.rb [hunk 1]
```diff
@@ -16,26 +16,12 @@ def analytics_report_json
...
       start_date: report_since.to_date,
       end_date: Date.today,
-      ctas: cta_json,
       visits: visits_json,
       users: users_json,
       macros: macros_json
     }
   end
 
-  def cta_json
```
Comment (donrestarone) [inline app/services/analytics_report_service.rb:26]: @Pralish can you spec out/estimate how we can add analytics to API namespaces/resources? This is not a top priority but I'd like to have it done before the Violet Rails 1.0 release

**Comment by donrestarone** [comment]
@Pralish I have notified clients that this deprecation is coming and they have all migrated onto API. Can you fix the merge conflicts and go over the migration strategy and a quick regression run to make sure we didn't miss anything? 

I'll deploy this to staging early next week.

**Comment by Pralish** [comment]
> @Pralish I have notified clients that this deprecation is coming and they have all migrated onto API. Can you fix the merge conflicts and go over the migration strategy and a quick regression run to make sure we didn't miss anything?
> 
> I'll deploy this to staging early next week.

@donrestarone , Have they manually ran the migration script? Because the migration script didn't make any changes for me because of this condition when I ran the migration. 


> if models.include?("CallToAction")

Since we are removing the CTA model and running the migration in the same PR, the migration script will never execute because the CTA model won't be there anymore. Was this intentional?

Thanks

**Pull request merged**